src/Symfony/Bridge/Monolog/Handler/FingersCrossed/HttpCodeActivationStrategy.php

```diff
@@ -19,17 +20,27 @@
  * Activation strategy that ignores certain HTTP codes.
  *
  * @author Shaun Simmons <shaun@envysphere.com>
+ * @author Pierrick Vignand <pierrick.vignand@gmail.com>
  */
-class HttpCodeActivationStrategy extends ErrorLevelActivationStrategy
+/* final */ class HttpCodeActivationStrategy extends ErrorLevelActivationStrategy implements ActivationStrategyInterface
```

Comment:
You should use a `@final` phpdoc instead.

Reply:
done